plugins/premium/tinymcespellchecker.md

```diff
@@ -343,6 +343,32 @@ tinymce.init({
 });
 ```
 
+### SpellcheckerLanguageChanged event
+
+This event fires when the spellchecking language is changed.
+
+{{ site.requires_5_7v }}
```

Comment:
This should be above the text I'm pretty sure, or at least that's how it's done elsewhere :shrug: